src/main/java/com/commercetools/sync/producttypes/ProductTypeSync.java

```diff
@@ -226,20 +223,9 @@ private void handleError(@Nonnull final String errorMessage, @Nullable final Thr
             .stream()
             .map(newProductType ->
                 removeAndKeepTrackOfMissingNestedAttributes(newProductType, keyToIdCache))
-            .map(draftWithoutMissingRefAttrs ->
-                referenceResolver
-                    .resolveReferences(draftWithoutMissingRefAttrs)
-                    .thenCompose(resolvedDraft -> syncDraft(oldProductTypeMap, resolvedDraft))
-                    .exceptionally(completionException -> {
```

Comment:
I removed the exceptionally part, because I don't expect any error to occur at this point. If any occur, then it must be programmatical error or a case that we didn't think of.